test/integration/forms/action.js

```diff
@@ -2113,7 +2113,7 @@ context('Patient Action Form', function() {
       .wait('@routeFormDefinition');
 
     const errors = getErrors({
-      status: '403',
+      status: 403,
```

Comment:
status is a string I believe and `getErrors` forces it to be a string..

Reply:
Ah didn't realize that... and now they are different everywhere in the tests. Some are a string and some are a number. I'm going to switch all of them to a string so they are all the same.

Reply:
I just switched those over to all be strings in a new commit: 9755d453c9bbaeb44cbc3cb09aa6109e435ebf99.